packages/env/lib/build-docker-compose-config.js

```diff
@@ -176,9 +176,17 @@ module.exports = function buildDockerComposeConfig( config ) {
 				image: 'mariadb',
 				ports: [ '3306' ],
 				environment: {
-					MYSQL_ALLOW_EMPTY_PASSWORD: 'yes',
+					MYSQL_ROOT_PASSWORD: 'password',
+					MYSQL_DATABASE: 'wordpress',
+				},
+			},
+			'tests-mysql': {
+				image: 'mariadb',
+				ports: [ '3306' ],
+				environment: {
+					MYSQL_ROOT_PASSWORD: 'password',
+					MYSQL_DATABASE: 'tests-wordpress',
 				},
-				volumes: [ 'mysql:/var/lib/mysql' ],
```

Comment:
The volume for the `mysql` container should be retained. Removing this means any restart of the environment loses any existing data.

The `clean` command exists to wipe databases on the dev environment when needed.

Reply:
I think restarting the environment doesn't mean losing data, destroying the environment means that and for a dev environment, I think that's totally acceptable.

Reply:
Regardless, I restored volumes to see if it solves the permission issues